lib/adapter/freshheads/CssMinimizationAdapter.ts

```diff
@@ -0,0 +1,33 @@
+import { Adapter, NextCallback } from '../Adapter';
+import { Configuration } from 'webpack';
+import { BuilderConfig } from '../../Builder';
+import { OptimizationAdapter } from '../..';
+import { validateIfRequiredModuleIsInstalled } from '../../utility/moduleHelper';
+
+export default class CssMinimizationAdapter implements Adapter {
+    public apply(
+        webpackConfig: Configuration,
+        builderConfig: BuilderConfig,
+        next: NextCallback
+    ) {
+        validateIfRequiredModuleIsInstalled(
+            'CssMinimizationAdapter',
+            'css-minimizer-webpack-plugin',
+            '1.2.0'
+        );
+
+        // Minimizer will only be run in production mode so we don't need to check it
```

Comment:
Ik zie hier nergens een check dat dit stukje code alleen in productie-mode runt, toch? Of mis ik iets?

Reply:
Optimizer / minimizers zijn afhankelijk van je mode config setting. En worden standaard alleen in de production mode gerunt. Je kan hem wel forceren om ook in de develop te runnen door hier `minimize: true` te zetten. Maar dat willen we niet. Heb het getest en het werkt.

Reply:
zie ook: https://webpack.js.org/plugins/css-minimizer-webpack-plugin/

Reply:
ok